### PR TITLE
Issue - Videos, Mobile Nav, Reset password modal

### PR DESCRIPTION
**Pull request by mhfen**
### What's this PR do?
- Fixes video encoding to allow IE11 to play videos
- Fixes overflow on mobile nav
- Fixes reset password modal
- Fixes login redirect in IE11
- Auto-updates the Current year in copyright footer

### What are the relevant tickets?
https://mondorobot.atlassian.net/browse/RET-221
https://mondorobot.atlassian.net/browse/RET-350
https://mondorobot.atlassian.net/browse/RET-355

### Who should review this PR?
@cprater @mhfen

### CODE REVIEW

**Comment by mhfen** [comment]
@cprater This is ready

File: src/assets/styles/modules/landmarks/nav/nav-drawer/_nav-drawer.scss [hunk 1]
```diff
@@ -19,11 +21,11 @@
 }
 
 .navDrawer-inner {
-  left: 0;
-  position: absolute;
-  top: 50%;
+  max-height: 100%;
+  max-height: 100vh;
```
Comment (cprater) [inline src/assets/styles/modules/landmarks/nav/nav-drawer/_nav-drawer.scss]: two `max-height`s?

File: src/client/pages/auth/reset-password.jsx [hunk 2]
```diff
@@ -60,19 +62,34 @@ export default class ResetPassword extends Component {
...
+      this.setState({
+        'serverError': false,
+        'serverMessages': []
+      });
       this.save();
     }
   }
 
+  formHasErrors() {
+    let errors = this.fields.filter(field => {
```
Comment (cprater) [inline src/client/pages/auth/reset-password.jsx]: this can be a `const`

File: src/client/pages/auth/reset-password.jsx [hunk 3]
```diff
@@ -60,19 +62,34 @@ export default class ResetPassword extends Component {
...
+      const hasError = field.state.isError || field.input.value.trim().length === 0;
+
+      if (hasError) {
```
Comment (cprater) [inline src/client/pages/auth/reset-password.jsx]: Do we want to `setState` for every error, or should we just set the state once after this filter if any errors exist?

File: src/client/pages/auth/reset-password.jsx (hunk 2 above)
Comment (cprater) [inline src/client/pages/auth/reset-password.jsx]: Actually we might could use `Array.some()` here instead of `filter`

**Pull request merged**